src/create-context.js

```diff
@@ -30,7 +30,19 @@ export function createContext(defaultValue) {
 		};
 		comp.componentDidUpdate = () => {
 			let v = comp.props.value;
-			subs.map(c => v!==c.context && (c.context = v, enqueueRender(c)));
+			subs.map(c => {
+				// A component can have multiple `useContext` hooks
+				if (c.contextType==null && c.__hooks) {
+					c.__hooks._list.forEach(state => {
```

Comment:
(I'm trying to think of ways to avoid pulling the hooks-specific part of this into core.)

Reply:
Yeah, it doesn't feel right :/